.github/workflows/contracts.yml

```diff
@@ -189,3 +189,62 @@ jobs:
       #     token: ${{ secrets.NPM_TOKEN }}
       #     access: public
       #     dry-run: true # TODO: to be removed once testing of workflow is done
+
+  migrate-and-publish-celo:
+    needs: [build-and-test, lint]
+    if: |
+      github.ref == 'refs/heads/master'
+        && (github.event_name == 'push'
+        || github.event_name == 'workflow_dispatch')
+    environment: keep-test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    defaults:
+      run:
+        working-directory: ./solidity
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-solidity-node-modules
+        with:
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Migrate contracts
+        env:
+          CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY: |
+            ${{ secrets.KEEP_TEST_CELO_CONTRACT_OWNER_PRIVATE_KEY }}
```

Comment:
@nkuba: In `migrate-and-publish-ethereum` job we use this variable:
`CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY: ${{ secrets.CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY }}`
Maybe it would be good to change its name to `KEEP_TEST_ETH_CONTRACT_OWNER_PRIVATE_KEY` to make it  similar to what we use here for Celo?
We could change that as part of https://github.com/keep-network/keep-core/pull/2376 or leave it for later and change it with other variables that also may need some name changes (like adding `KEEP_TEST_` prefix).

Reply:
Yep @michalinacienciala, that was something I was thinking about when adding the celo secret.

Reply:
`KEEP_TEST_` prefix has been added now to several variables:
```KEEP_TEST_ETH_CONTRACT_OWNER_PRIVATE_KEY
KEEP_TEST_ETH_HOSTNAME
KEEP_TEST_ETH_NETWORK_ID
KEEP_TEST_ETH_TRUFFLE_NETWORK```
(changes done both in GitHub settings and in `contracts.yml`).